### DISCUSSION TITLE
Using Jamulus on a LAN-network, not connected to internet, is it possible?

### DISCUSSION THREAD

**Discussion by ZingenmetJan** [jamulussoftware/jamulus#3225] [Ideas]
Would it be possible to run jamulus on an intranet?
I am considering using Jamulus to have several singers or actors have their raspberry connected with wifi to a router. Each RPI will have a microphone and all those signals come together in a server, connected to this intranet (but not to internet). I will use wifi 5 GHz to prevent signals of a lot of other devices interfering and a router with a big capacity. I will use one PC or tablet to mix the signal and send it to the speakers to the audience.
Would this be possible or is there another way to do this?
I want to try this, because wireless equipment for 30 singers/actors is very expensive.

**Comment by chrisrimple** [comment]
Running a local Jamulus server and local Jamulus clients, with no connection to the Internet, should work fine.

But, running Jamulus clients over wifi is likely to introduce a lot of "jitter", which will cause latency and other issues, making it hard for the singers to remain in sync and to blend.  Even 5GHz wifi is unlikely to provide you with the experience you're seeking.

**Comment by rdica** [comment]
It's possible yes. Feasible? Probably not without making some sacrifices.

Will each participant be remote?
You mention use of a LAN.
Is each participant in a separate room in the same building?
MUST they use wireless?

Wireless will introduce higher jitter resulting in audio dropouts and other network related audio artifacts.

If the participants are relying on jamulus to sync with each other, that will be more difficult due to the increased latency and jitter that wireless connectivity introduces.

**Comment by dcorson-ticino-com** [comment]
No problem, as long as you do not use wireless at any point.

You can try wireless, but you will be disappointed. 
5G is not better than 2.4G.
The problems you will see/hear are because of the wireless and not Jamulus.

**Comment by pcar75** [comment]
Also  

> (...) send it to the speakers to the audience. (...)

If the 'audience' (singers ?, players ? ) has microphones your will get unworkable feedback !

**Comment by ann0see** [comment]
If you need wireless and are all in a close area, look at professional wireless mics and headphones + a mixer. I think that would give a much better experience
